### PR TITLE
CORTX-33695 Add support for configurable instance and volume count in terraform codebase

### PR DESCRIPTION
**Pull request by shailesh-vaidya**
# Problem Statement
- Improve code base to take Instance count and volume count as an input value. 

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [x] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [ ] Testing was performed with RPM
- [ ] Jenkins pipelines used for testing

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

File: solutions/community-deploy/cloud/AWS/main.tf [hunk 1]
```diff
@@ -154,11 +158,14 @@ resource "aws_instance" "cortx_deploy" {
 }
 
 resource "aws_ebs_volume" "data_vol" {
-  count             = 9
+  count             = var.ebs_volume_count * var.instance_count
   availability_zone = data.aws_availability_zones.available.names[0]
   size              = 10
+  tags = {
```
Comment (Abhirup-A-G) [inline solutions/community-deploy/cloud/AWS/main.tf:164]: Have you tested with this tag ? as i have also tried similar thing but on the name column was still showing blank for me.

Comment (shailesh-vaidya) [reply solutions/community-deploy/cloud/AWS/main.tf:164]: Yes. Volumes are getting created with proper names 

image

**Review by Abhirup-A-G — APPROVED** [review]
Approved.

**Review by mukul-seagate11 — APPROVED** [review]
looks good & improve on the hardcoded values as improvement

**Review by nitisdev — APPROVED** [review]

**Pull request merged**